### PR TITLE
Media wrapper for showing videos

### PR DESCRIPTION
**Pull request by liviaw**
add encapsulation for showing videos with templates

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,9 @@
 /.pnp
 .pnp.js
 
+yarn.lock
```
Comment (juliamitchelmore) [inline .gitignore]: 👍

File: src/App.tsx [hunk 2]
```diff
@@ -4,6 +4,9 @@ import { ImportModal } from "./Components/ImportModal/ImportModal";
 import { Header } from "./Components/Shared/Header";
 import { Container } from "react-bootstrap";
 import { MediaPresenter } from "./Components/MediaPresenter";
+import { VideoModal } from "./Components/VideoModal/VideoModal";
+
+
```
Comment (juliamitchelmore) [inline src/App.tsx]: nit: watch your whitespace!

File: .gitignore (hunk 1 above)
Comment (juliamitchelmore) [reply .gitignore]: although `yarn.lock` is still committed here... ??
You might need to add the gitignore change in a separate PR

**Review by juliamitchelmore — APPROVED** [review]
LGTM if you can sort out the yarn.lock issue

**Comment by liviaw** [comment]
no file changed

**Comment by liviaw** [comment]
no file changed

**Pull request closed without merging**